.github/workflows/ci.yml

```diff
@@ -39,7 +43,13 @@ jobs:
     - name: Install pymgclient
       run: python3 -m pip install ./dist/pymgclient-*
     - name: Run tests
-      run: MEMGRAPH_PORT=10000 python3 -m pytest
+      run: |
+        MEMGRAPH_PORT=10000
+        if [ ${{ matrix.mgversion }} == "2.0.0" ]; then
```

Comment:
Maybe use `>=` to make sure temporal type tests will run also in the future versions? Also below in the windows run.